diana.navasardyan/repo/Lesson_6/exercise_07/exercise_07.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+num_1 = int(input(" Please enter a number (Hint:Enter '0' to calculate the average): "))
```

Comment:
LE: The user should enter at least one number before entering 0. Use hint after the user can enter 0.